terminus-ui/button/src/button.component.ts

```diff
@@ -157,8 +154,7 @@ export class TsButtonComponent implements OnInit, OnDestroy {
 
     // Verify the value is allowed
     if (tsButtonFormatTypesArray.indexOf(value) < 0 && isDevMode()) {
-      console.warn(`TsButtonComponent: "${value}" is not an allowed format. ` +
-      `See TsButtonFormatTypes for available options.`);
+      console.warn(`TsButtonComponent: "${value}" is not an allowed format. See TsButtonFormatTypes for available options.`);
```

Comment:
I've gone back and forth a bit on this.. in general I went with logs simply because they don't completely stop compilation and most of our warnings are around using a component correctly. But usually we don't need to stop all the things.

We're not 100% consistent on this so we'll need to make a more firm decision at some point.